tree/dataframe/src/RDFBookedCustomColumns.cxx

```diff
@@ -4,23 +4,24 @@ namespace ROOT {
 namespace Internal {
 namespace RDF {
 
-bool RBookedCustomColumns::HasName(std::string name) const
+bool RBookedCustomColumns::HasName(std::string_view name) const
 {
-   return std::find(fCustomColumnsNames->begin(), fCustomColumnsNames->end(), name) != fCustomColumnsNames->end();
+   const auto ccolnamesEnd = fCustomColumnsNames->end();
+   return ccolnamesEnd != std::find(fCustomColumnsNames->begin(), ccolnamesEnd, name);
```

Comment:
would it be better in general to search in `fCustomColumns` (a `std::map`) rather than in `fCustomColumnsNames` (a `std::vector`)?

Reply:
of course.